### PR TITLE
Navigation and modal components

### PR DESCRIPTION
**Pull request by samyukthasudhakar**
Created the following components as part of this PR :

- Navigation Bar
-  Modal

Live : https://nav-and-modal.netlify.app/docs.html#nav

### CODE REVIEW

**Review by Maianki — APPROVED** [review]
Looks good to merge.

File: docs.html [hunk 1]
```diff
@@ -669,6 +671,93 @@ <h3>Text only card example</h3>
...
+                    style="width: 742px; height: 524px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </div>
+            </section>
+
+            <!-- Modal Component -->
+            <section id="Modal" class="component-section">
```
Comment (partha8) [inline docs.html]: you have capitalised M here, "Modal".

File: styles/components/modal.css [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    left:50%;
+    transform: translate(-50%, -50%);
+    display:none;
+}
+
+#modal-cancel, .modal-cancel{
+    border:none;
+    background-color:transparent;
+    align-self:flex-end;
+}
```
Comment (partha8) [inline styles/components/modal.css]: Don't use id selectors as they might cause problems. I am attaching a blog you can refer it for better understanding. 

https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4#:~:text=IDs%20have%20a%20much%20higher,specificity%20wars%20in%20your%20stylesheets.

**Review by Dev-AkshatGupta — APPROVED** [review]
Looks great to me Samuyuktha a small suggestion is to have a bit slow transitioning in the aside nav

**Review by ArjunJunna — APPROVED** [review]
For better UI in the aside menu, you can increase its width to 20vw.  Rest everything looks good.

File: styles/components/modal.css (hunk 2 above)
Comment (samyukthasudhakar) [reply styles/components/modal.css]: > Don't use id selectors as they might cause problems. I am attaching a blog you can refer it for better understanding.
> 
> https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4#:~:text=IDs%20have%20a%20much%20higher,specificity%20wars%20in%20your%20stylesheets.

Thanks for mentioning this. Have made the required changes. The article was helpful 😄

**Comment by samyukthasudhakar** [comment]
> Looks great to me Samuyuktha a small suggestion is to have a bit slow transitioning in the aside nav

Sure will do that, thank you.

File: docs.html (hunk 1 above)
Comment (samyukthasudhakar) [reply docs.html]: Changed it to lower case.

**Comment by samyukthasudhakar** [comment]
> For better UI in the aside menu, you can increase its width to 20vw. Rest everything looks good.

Sure. Thank you.

**Review by partha8 — APPROVED** [review]

**Pull request merged**